test/2d.jl

```diff
@@ -334,10 +334,10 @@ end
 
     err = ArgumentError("$(typestring(Fill{Int,1}))(0, (3,), (3,)) lacks the proper padding sizes for an array with 2 dimensions")
     kern = Kernel.gaussian((1,1),(3,3))
-    @test_throws err imfilter(A, kern, Fill(0, (3,)))
+    @test_throws err imfilter(CPU1(), A, kern, Fill(0, (3,)))
     kernf = ImageFiltering.factorkernel(kern)
-    err = DimensionMismatch("requested indices (1:8, 0:9) and kernel indices $(axes(kernf[1])) do not agree with indices of padded input, (0:9, 1:8)")
-    @test_throws err imfilter(A, kern, Fill(0, (1,0)))
-    @test_throws DimensionMismatch imfilter(A, kern, Fill(0, (0,1)))
-    @test_throws DimensionMismatch imfilter(A, kern, Fill(0, (0,0)))
+    err = DimensionMismatch("output indices (OffsetArrays.IdOffsetRange(0:9), OffsetArrays.IdOffsetRange(1:8)) disagrees with requested indices (1:8, 0:9)")
```

Comment:
```suggestion
    err = DimensionMismatch("output indices (OffsetArrays.IdOffsetRange(0:9), OffsetArrays.IdOffsetRange(1:8)) disagree with requested indices (1:8, 0:9)")
```

("indices" is plural...parenthetically, English is **so** weird in that plural nouns get an `s` but verb agreement drops the `s`!)